LeanKit-Labs/tag-release
# Feature pr deux

### Short description of the work completed

> Typically when running `--pr` we hit conflicts in the package.json that should able to automatically resolve since we should only take the changes we did locally, aka our pre-releases, and accept anything that changed in HEAD. This PR does just that!

### Steps to test (if not obvious)

1. Create a branch based off of master and go through the `--qa` flow.
2. Create another branch based off of master and go through the `--qa` flow and change things in the same spot you did previously.
3. Merge in the first branch using `--pr`, this should resolve with no conflicts.
4. Merge in the second branch using the `--pr` flag, this should hit conflicts but we able to be automatically resolved by the added functionality of this PR.


### For Reviewer Use Only

  patch)
